python/MetrologyData.py

```diff
@@ -342,29 +274,6 @@ def plot_statistics(self, nsigma=5, title=None, zoffset=0):
         y = y/np.sum(y)*np.size(dz)*binsz/0.1
         plot.pylab.plot(x, y, color='b', linestyle='-')
 
-        # Overlay the model residuals
```

Comment:
I should have read further.  Yes, I see that lim can be omitted, and will take your recommendation below.